src/openforms/registrations/contrib/zgw_apis/models.py

```diff
@@ -57,6 +58,32 @@ class ZgwConfig(SingletonModel):
         validators=[validate_rsin],
         help_text=_("Default RSIN of organization, which creates the ZAAK"),
     )
+    zaak_vertrouwelijkheidaanduiding = models.CharField(
+        _("vertrouwelijkheidaanduiding zaak"),
+        max_length=24,
+        choices=VertrouwelijkheidsAanduidingen.choices,
+        default=VertrouwelijkheidsAanduidingen.zaakvertrouwelijk,
+        help_text=_(
+            "Indication of the level to which extend the ZAAK is meant to be public. "
+            "Can be overridden in the Registration tab of a given form."
+        ),
+    )
+    doc_vertrouwelijkheidaanduiding = models.CharField(
+        _("vertrouwelijkheidaanduiding document"),
+        max_length=24,
+        choices=VertrouwelijkheidsAanduidingen.choices,
+        default=VertrouwelijkheidsAanduidingen.zaakvertrouwelijk,
+        help_text=_(
+            "Indication of the level to which extend the document associated with the "
+            "ZAAK is meant to be public. Can be overridden in the file upload "
+            "component of a given form."
+        ),
+    )
```

Comment:
these should be `blank=True` without a default set, since the default is baked into the `zaaktype` and `informatieobjecttype` for the zaak/document being created. The Zaken API/Documenten API have this business logic baked in.

Checked this with Joeri - the ticket spec is ambiguous/wrong here.